src/test/testng-ci.xml

```diff
@@ -50,6 +50,7 @@
     <test name="parallel-ui-tests" parallel="classes">
         <!-- These cover the important UI functionality. Can be run in parallel. -->
         <classes>
+            <class name="teammates.test.cases.browsertests.GodModeTest" />
```

Comment:
I was using it to verify that the MOTD URL is defined, though on second thoughts, that's not really necessary. I also just realised that this PR allows GodMode enabled on Travis without failing the build 😅 Fixing.